mkosi/__init__.py

```diff
@@ -2241,6 +2242,56 @@ def install_openmandriva(args: CommandLineArguments, root: str, do_run_build_scr
     disable_pam_securetty(root)
 
 
+@complete_step("Installing Gentoo")
+def install_gentoo(args: CommandLineArguments, root: str, do_run_build_script: bool) -> None:
+    try:
+        import portage
+    except ImportError:
+        die("you need portage module for Gentoo: https://gitweb.gentoo.org/proj/portage.git")
+
+    from _emerge.actions import load_emerge_config
+    from _emerge.actions import action_info
+    from _emerge.actions import action_sync
+    from _emerge.actions import action_build
```

Comment:
The `portage` import doesn't seem to be needed? If you don't, leave it out and move the imports from `_emerge` into the try block.

Also, move all the `_emerge` imports into a single statement.